open-redirect.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+- id: open-redirect
+  mode: taint
+  pattern-sources:
+  - patterns:
+    - pattern: $REDIRECTURL
```

Comment:
Thanks for the contribution! By pattern-sources, we usually want to specify a function call where the user can input things (for instance if it's an argument to a function definition, you can do something like [this](https://github.com/returntocorp/semgrep-rules/blob/18f52b67a7a41e5cbfa94a22ec569185515d2966/csharp/dotnet/security/razor-template-injection.yaml).